editoast/src/views/timetable/similar_trains.rs

```diff
@@ -101,20 +101,25 @@ pub(in crate::views) struct Request {
 #[derive(Debug, Serialize, ToSchema)]
 #[cfg_attr(test, derive(Deserialize, PartialEq))]
 struct SimilarTrainItem {
-    /// Both `train_name` and `start_time` are `None` if no similar train
-    /// was found for the segment; otherwise, both are `Some`.
-    #[schema(required)]
-    train_name: Option<String>,
-    /// Both `train_name` and `start_time` are `None` if no similar train
-    /// was found for the segment; otherwise, both are `Some`.
-    #[schema(required)]
-    start_time: Option<DateTime<Utc>>,
+    #[schema(inline)]
+    /// `train` is `None` if no similar train
+    /// was found for the segment; otherwise, it is `Some`.
+    train: Option<Train>,
     #[schema(value_type = String)]
     begin: ArcStr,
     #[schema(value_type = String)]
     end: ArcStr,
 }
 
+#[derive(Debug, Serialize, ToSchema)]
+#[cfg_attr(test, derive(Deserialize, PartialEq))]
+struct Train {
+    #[schema(required)]
+    train_name: String,
+    #[schema(required)]
```

Comment:
```suggestion
    train_name: String,
```

I believe these aren't needed since the types aren't Options

Reply:
Done